composer.json

```diff
@@ -1,46 +1,49 @@
 {
-  "name": "prestashop/ps_facetedsearch",
-  "description": "PrestaShop module ps_facetedsearch",
-  "homepage": "https://github.com/PrestaShop/ps_facetedsearch",
-  "license": "AFL-3.0",
-  "authors": [
-    {
-      "name": "PrestaShop SA",
-      "email": "contact@prestashop.com"
-    }
-  ],
-  "require": {
-    "php": ">=5.6",
-    "doctrine/collections": "^1.4"
-  },
-  "require-dev": {
-    "friendsofphp/php-cs-fixer": "^2.14",
-    "phpunit/phpunit": "~5.7",
-    "prestashop/php-coding-standards": "dev-master",
-    "mockery/mockery": "^1.2"
-  },
-  "config": {
-    "platform": {
-      "php": "5.6.0"
+    "name": "prestashop/ps_facetedsearch",
```

Comment:
Indented but I think it's harmess

Reply:
I even don't know how the indentation was changed (I only used header-stamp :facepalm: )

Reply:
It's a known bug of header-stamp. It decodes the entire JSON, adds its stamp, then use `file_put_contents(json-encode($content));` to put content back into composer.json in so the indent used will be the one used on the machine (dont know how it's chosen).

Reply:
:facepalm: